src/RoleObjectFactory.js

```diff
@@ -164,12 +165,12 @@ function createAccessibilityObjectForRole(config) {
       accessibilityObject = new GroupData(displayObject, role, domIdPrefix);
       break;
 
-    case ROLES.HEADING1:
-    case ROLES.HEADING2:
-    case ROLES.HEADING3:
-    case ROLES.HEADING4:
-    case ROLES.HEADING5:
-    case ROLES.HEADING6:
+    case ROLES.HEADING1: // eslint-disable-line
+    case ROLES.HEADING2: // eslint-disable-line
+    case ROLES.HEADING3: // eslint-disable-line
+    case ROLES.HEADING4: // eslint-disable-line
+    case ROLES.HEADING5: // eslint-disable-line
+    case ROLES.HEADING6: // eslint-disable-line
```

Comment:
Please add the particular rule(s) that need to be disabled instead of disabling all linting. It helps others understand what the linting issue is for these lines (though I'm guessing it's no-fallthrough here).

Reply:
case ROLES[`HEADING${1}`]: is this ok to concat?